experimental/design-notes.txt

```diff
@@ -0,0 +1,112 @@
+I started comparing OpenTelemetry metrics to other metric APIs and wanted to jot down observations and questions that came up along the way:
+
+Micrometer (MM) - https://micrometer.io/docs/concepts
+
+1. Do the names that OpenTelemetry chose match industry conventions? 
```

Comment:
I'm wondering maybe we leave the "naming" convention match to the SDK.  Thus, API just needs to collect all the component fields, and let the SDK do what they want with forming a correct "name" for the exporter.

Basically, I'm saying maybe we are not so concern with naming from API side as it can be adjusted by the SDK.